fvtest/porttest/CMakeLists.txt

```diff
@@ -162,13 +162,60 @@ if (OMR_OS_ZOS AND OMR_ENV_DATA64)
 	)
 endif()
 
-if (NOT (OMR_OS_AIX OR OMR_OS_ZOS OR CMAKE_CROSSCOMPILING))
-	omr_add_test(
-		NAME porttest
-		COMMAND $<TARGET_FILE:omrporttest> --gtest_output=xml:${CMAKE_CURRENT_BINARY_DIR}/omrporttest-results.xml
-	)
+if(OMR_OS_AIX)
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}-PortSockTest.two_socket_datagram_communication")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSlTest.sl_AixDLLMissingDependency")
+elseif(OMR_OS_ZOS)
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}-PortDumpTest.dump_test_create_dump_with_no_path")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test20")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test21")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test22")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test23")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test24")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test25")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test27")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileStreamTest.omrfilestream_test_text")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileStreamTest.omrfilestream_test_omrfile_text_comparison")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSigTest.sig_test_async_handler")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSigTest.sig_test_single_async_handler")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSigTest.sig_test_mix_async_handler")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSigTest.sig_test_mix_two_async_handlers")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_test2")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_convPlatTo8")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_convU8ToPlat")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_convU8ToWide")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_convWideToU8")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortStrTest.str_convU8ToPlat_Null")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSysinfoTest.sysinfo_test_get_cwd3")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortSockTest.two_socket_datagram_communication")
+elseif(OMR_OS_LINUX AND OMR_ARCH_RISCV AND CMAKE_CROSSCOMPILING)
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}-PortFileTest2.file_test20")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test21")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test22")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test23")
+	set(EXCLUDE_LIST "${EXCLUDE_LIST}:PortFileTest2.file_test24")
```

Comment:
Each `if/elseif` block corresponds to a platform/spec in order to keep the exclude-list easy to read. This leads to duplicates. Avoiding the duplicates will lead to extra logic, which will make it difficult to comprehend the exclude-list.